### ISSUE TITLE
Add Guard Rails for sloped/inclined roads

### ISSUE THREAD

**Issue by jemmotion** [StrangeLoopGames/EcoSuggestions#1630]
Please add the function for guard rails for cement and metal to go on inclined/sloped roads at an incline obviously not floating in mid air.

**Comment by MishkaFur** [comment]
Corrugated Steel guard rail form like the asphalt ramp "1x4" perhaps? 

![image](https://user-images.githubusercontent.com/61889138/162586854-6afac3d5-7e81-4871-b18a-fa0749026626.png)

**Comment by jemmotion** [comment]
> Corrugated Steel guard rail form like the asphalt ramp "1x4" perhaps?
> 
> ![image](https://user-images.githubusercontent.com/61889138/162586854-6afac3d5-7e81-4871-b18a-fa0749026626.png)

Yeah that could work making a corrugated steel guard into a separate 1x4 ramp or could the the corrugated steel guard rail be smart enough to sit on the inclined ramp at an incline when in build mode? That way you wouldn't need a whole new object?
